lib/solargraph/rbs_map/conversions.rb

```diff
@@ -29,11 +29,37 @@ def type_aliases
         @type_aliases ||= {}
       end
 
+      # @param loader [RBS::EnvironmentLoader]
+      # @return [void]
+      def load_environment_to_pins(loader)
+        environment = RBS::Environment.from_loader(loader).resolve_type_names
+        cursor = pins.length
+        environment.declarations.each { |decl| convert_decl_to_pin(decl, Solargraph::Pin::ROOT_PIN) }
+        added_pins = pins[cursor..-1]
+        add_back_implicit_pins(added_pins)
+      end
```

Comment:
The definition of a class may be split among different declaration files.  As a result, we were prematurely creating implicit initializers, resulting in multiple initializers defined as well as one of them having the wrong arity.